### PR TITLE
Add params for test configuring

### PR DESCRIPTION
**Pull request by maccelf**

### CODE REVIEW

File: alts/shared/models.py [hunk 1]
```diff
@@ -26,11 +26,18 @@ class AsyncSSHParams(BaseModel):
...
+    tests: typing.Optional[typing.List[dict]]
+    test_env: typing.Optional[dict]
+
+
 class TaskRequestPayload(BaseModel):
     runner_type: str = 'any'
     dist_name: str
     dist_version: typing.Union[str, int]
     dist_arch: str
+    package_channel: typing.Optional[str] = None
```
Comment (anfimovdm) [inline alts/shared/models.py:39]: you can get rid of validation for `package_channel` field if you do the following:
```python
package_channel: typing.Optional[typing.Literal[*constants.ALLOWED_CHANNELS]]
```

File: alts/shared/models.py [hunk 2]
```diff
@@ -212,6 +232,8 @@ def __init__(self, **data):
     opennebula_username: typing.Optional[str]
     opennebula_password: typing.Optional[str]
     opennebula_vm_group: typing.Optional[str]
+    opennebula_network: typing.Optional[str]
+    allowed_channel_names: typing.List = ['stable', 'beta']
```
Comment (anfimovdm) [inline alts/shared/models.py:236]:
```python
allowed_channel_names: typing.List[str] = constants.ALLOWED_CHANNELS
```

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alts/shared/constants.py [hunk 3]
```diff
@@ -15,8 +16,10 @@
 SUPPORTED_DISTRIBUTIONS = ['almalinux', 'centos', 'ubuntu', 'debian']
 RHEL_FLAVORS = ('rhel', 'fedora', 'centos', 'almalinux', 'cloudlinux')
 DEBIAN_FLAVORS = ('debian', 'ubuntu', 'raspbian')
+ALLOWED_CHANNELS = ['stable', 'beta']
 
 
 DEFAULT_FILE_CHUNK_SIZE = 8388608  # 8 megabytes in bytes
 DEFAULT_REQUEST_TIMEOUT = 60  # 1 minute
 DEFAULT_UPLOADER_CONCURRENCY = 4
+DEFAULT_TEST_TIMEOUTS = {} # Default test execution timeouts
```
[the remark this answers is not in the document]
Comment (maccelf) [reply alts/shared/constants.py:25]: This constant for the future

[the remark this answers is not in the document]
Comment (maccelf) [reply alts/shared/constants.py:25]: NB:

Default test execution timeouts should be hardcoded in scheduler and workers.

[the remark this answers is not in the document]
Comment (Korulag) [reply alts/shared/constants.py:25]: Not sure about hard-coding, yet we need to have a timeout for each script/command execution that happens during package testing process (executor-level timeouts) and make sure they are honored.

File: alts/shared/models.py (hunk 1 above)
Comment (maccelf) [reply alts/shared/models.py:39]: > Unpacked arguments cannot be used in type argument lists
> Unpack operator in subscript requires Python 3.11 or newer

`Pyright`

File: alts/shared/constants.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (maccelf) [reply alts/shared/constants.py:25]: Ok, I'll remove it

File: alts/shared/models.py (hunk 1 above)
Comment (anfimovdm) [reply alts/shared/models.py:59]: you're right, but anyway, you can get rid of validation using `typing.Literal['stable', 'beta']`

**Review by anfimovdm — CHANGES_REQUESTED** [review]

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**